include/robot_interfaces/sensor_data_types.hpp

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include <cmath>
+#include <ctime>
+#include <real_time_tools/threadsafe/threadsafe_timeseries.hpp>
+#include <time_series/time_series.hpp>
+
+#include <opencv2/opencv.hpp>
+
+#include <robot_interfaces/opencv_driver.hpp>
+#include <robot_interfaces/sensor_backend.hpp>
+#include <robot_interfaces/sensor_data.hpp>
+#include <robot_interfaces/sensor_frontend.hpp>
+
+namespace robot_interfaces
+{
+/**
+ * @brief Collection of data types for the sensors used in the trifinger
+ * platform.
+ *
+ * Refer to the docstrings of individual data types for more details.
+ */
+
+struct SensorDataTypes
```

Comment:
As this is specifying the types for the OpenCV camera, it should rather be called `OpenCVCameraTypes` or maybe better just `CameraTypes` as any other camera driver would probably also use cv::Mat to store the frame, no matter how it is grabbed.

Reply:
After thinking a bit more about it, I am actually unsure if we even need this struct at all. We could instead just define the Observation type and then specify the template argument everywhere when creating the frontend, backend, etc.

Reply:
Yeah I had thought about this too, then just stuck to the same structure. Changed now, thanks.